server/router/src/test/java/org/infinispan/server/router/utils/HotRodClientTestingUtil.java

```diff
@@ -0,0 +1,52 @@
+package org.infinispan.server.router.utils;
+
+import org.infinispan.client.hotrod.RemoteCacheManager;
+import org.infinispan.client.hotrod.configuration.ConfigurationBuilder;
+
+import java.net.InetAddress;
+
+public enum HotRodClientTestingUtil {
```

Comment:
@galderz Agree, the server stuff should be moved out of `HotRodTestingUtil`.

Still, it's a bit weird to have server-side tests depending on the HR client. I think it would be better to have these tests in the HR client module, so that the HR client depends on the router and not the other way around. The HR server module even has a dummy client in it to avoid depending on the client for the tests.